File: web/src/models/Evento.ts
export type TEvento =
{
   id?: number;
   nome: string;
   idUsuarioCriador: number;
   idEsporte: number;
   dtEvento: Date;
   descricao: string;
   local: string;

   nomeUsuario?: string;
   nomeEsporte?: string;
   qtdPresencas?: number;
   presente?: boolean;
}

File: api/src/main/java/com/faesa/api/model/Evento.java
package com.faesa.api.model;

import java.util.Date;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Evento
{
	private Integer id;
	private String nome;
	private Integer idUsuarioCriador;
	private Integer idEsporte;
	private String descricao;
	private Date dtEvento;
	private String local;
	private Date dtInsert;
	
	private String nomeUsuario;
	private String nomeEsporte;
	private Integer qtdPresencas;
	private Boolean presente;
}
